File: app/src/main/java/com/theroyalsoft/telefarmer/model/SimpleSdpObserver.java
package com.theroyalsoft.telefarmer.model;

import org.webrtc.SdpObserver;
import org.webrtc.SessionDescription;

import timber.log.Timber;

/**
 * Created by Pritom Dutta on 5/8/23.
 */
public class SimpleSdpObserver implements SdpObserver {
    @Override
    public void onCreateSuccess(SessionDescription sessionDescription) {
        Timber.i("SdpObserver: onCreateSuccess !");
    }

    @Override
    public void onSetSuccess() {
        Timber.i("SdpObserver: onSetSuccess");
    }

    @Override
    public void onCreateFailure(String msg) {
        Timber.e("SdpObserver onCreateFailure: " + msg);
    }

    @Override
    public void onSetFailure(String msg) {
        Timber.e("SdpObserver onSetFailure: " + msg);
    }
}

File: app/src/main/java/com/theroyalsoft/telefarmer/sockethelper/SimpleSdpObserver.kt
package com.theroyalsoft.telefarmer.sockethelper

import org.webrtc.SdpObserver
import org.webrtc.SessionDescription

/**
 * Created by Pritom Dutta on 9/9/23.
 */
class SimpleSdpObserver : SdpObserver {
    override fun onCreateSuccess(sessionDescription: SessionDescription?) {}
    override fun onSetSuccess() {}
    override fun onCreateFailure(s: String?) {}
    override fun onSetFailure(s: String?) {}
}
